### PR TITLE
Task/cp 39 cp 40 cp 41 enums

### PR DESCRIPTION
**Pull request by radu-diaconescu13**

### CODE REVIEW

File: project_template/datamodels/institute.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from enum import Enum
+
+
+class Institute(Enum):
```
Comment (RaduCStefanescu) [inline project_template/datamodels/institute.py]: Let's change this to "Institution"

File: project_template/datamodels/function.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from enum import Enum
+
+
+class Function(Enum):
```
Comment (RaduCStefanescu) [inline project_template/datamodels/function.py]: Let's change this to "Position"

Comment (radu-diaconescu13) [reply project_template/datamodels/function.py]: Done

File: project_template/datamodels/institute.py (hunk 1 above)
Comment (radu-diaconescu13) [reply project_template/datamodels/institute.py]: Done

**Pull request merged**